game/overlord/fake_iso.cpp

```diff
@@ -298,16 +303,60 @@ void FS_PollDrive() {}
 
 // TODO FS_LoadMusic
 uint32_t FS_LoadMusic(char* name, void* buffer) {
-  (void)name;
-  (void)buffer;
-  ASSERT(false);
+  s32* bank_handle = (s32*)buffer;
+  char namebuf[16];
+  strcpy(namebuf, name);
+  namebuf[8] = 0;
+  strcat(namebuf, ".mus");
+  auto file = FS_Find(namebuf);
+  if (!file)
+    return CMD_STATUS_FAILED_TO_OPEN;
+
+  *bank_handle = snd_BankLoadEx(get_file_path(file), 0, 0, 0);
+  snd_ResolveBankXREFS();
+
   return 0;
 }
 
 // TODO FS_LoadSoundBank
 uint32_t FS_LoadSoundBank(char* name, void* buffer) {
-  (void)name;
-  (void)buffer;
-  ASSERT(false);
+  SoundBank* bank = (SoundBank*)buffer;
+  char namebuf[16];
+
+  int offset = 10 * 2048;
+  if (bank->sound_count == 101) {
+    offset = 1 * 2048;
+  }
+
+  strcpy(namebuf, name);
+  namebuf[8] = 0;
+  strcat(namebuf, ".sbk");
+
+  auto file = FS_Find(namebuf);
+  if (!file) {
+    file = FS_Find("empty1.sbk");
```

Comment:
I think this is where the tests are failing, because we don't have empty1.sbk file available in the tests (we don't want to upload game files to github).  I haven't tried it yet, but it seems like it might be okay if you just did a `if (!file) return` here instead.

We don't run the game loop in tests, so there shouldn't try to play any sounds. But it does initialize the game code, which loads the common/empty1/empty2 banks.